src/Tmds.MDns/ServiceBrowser.cs

```diff
@@ -263,7 +259,18 @@ private void CheckNetworkInterfaceStatuses(Dictionary<int, NetworkInterfaceHandl
                         interfaceHandler = new NetworkInterfaceHandler(this, networkInterface);
                         _interfaceHandlers.Add(index, interfaceHandler);
                         OnNetworkInterfaceAdded(networkInterface);
-                        interfaceHandler.StartBrowse(_serviceTypes.Select(st => new Name(st.ToLower() + ".local.")));
+
+                        // Get the fully qualified name for each service type.  Normally, ".local."
+                        // is appended.  However, some services do not follow this convention.  So
+                        // if the service type ends with ".", it is considered a FQN.
+                        var names = _serviceTypes.Select(st =>
+                        {
+                            var fqn = st.ToLowerInvariant();
+                            if (!fqn.EndsWith("."))
+                                fqn = fqn + ".local.";
+                            return new Name(fqn);
+                        });
+                        interfaceHandler.StartBrowse(names);
```

Comment:
We shouldn't workaround js-libp2p-mdns behavior. It should be fixed in that project. Please remove this.

Reply:
Would prefer to keep it...but will remove it.